test/parallel/test-http-multiple-headers.js

```diff
@@ -64,6 +64,12 @@ const server = createServer(
       res.write('BODY');
       res.end();
 
+      assert.throws(() => res.appendHeader(), {
+        code: 'ERR_HTTP_HEADERS_SENT',
+        name: 'Error',
+        message: 'Cannot append headers after they are sent to the client'
```

Comment:
https://github.com/nodejs/node/blob/260092eca316bae6036e52b2cdc73c4c4f8a9da6/doc/contributing/writing-tests.md#L315-L325
```suggestion
```

Reply:
Oh, this is new one for me, thanks!